_docs/plugins/themes.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: documentation
+title: Themes
+category: Plugins
+order: 5.1
+---
+
+# Finding Themes
+
+Themes published on npm should have the keyword `lounge-theme`, so you can find a list by [searching on npmjs.org](https://www.npmjs.com/search?q=keywords%3Alounge-theme).
+
+# Installing Themes
+
+To install theme `lounge-theme-custom`
+
+* `npm install lounge-theme-custom`
+* Add `lounge-theme-custom` to the `plugins` array in `config.js`
+* Restart The Lounge
+* Select theme in options
+
+# Creating Themes
+
+Create an npm package using the base example.json below, and publish to npm.
+
+```
+{
+  "name": "lounge-theme-custom", // Replace 'custom' with your theme's name
+  "version": "1.0.0",
+  "description": "A theme for The Lounge",
+  "main": "package.json",
```

Comment:
Document why we do this, and explain that an actual JS file can be used to export.

Reply:
I did this by adding another section below, let me know what you think.

Reply:
Does this, referencing a JS file that exports an object, really work?
I believe [this](https://github.com/thelounge/lounge/blob/master/src/plugins/themes.js#L60-L69) does, but it would get stopped by [that](https://github.com/thelounge/lounge/blob/master/src/command-line/install.js#L26-L30) first if I understand correctly.

It's fairly common for libraries to use a key in the `package.json` like Babel, but I don't remember seeing them asking to export `package.json` or an object in JS, do you have an idea of how they access it?
Maybe I'm looking at this wrong, sorry if that's the case 😅